rigex-front-end/src/components/RigSidebar/sideBar.scss

```diff
@@ -68,9 +68,13 @@
   display: flex;
   align-items: center;
   cursor: pointer;
+  border-radius: 3px;
+}
+
+.rigListItem:hover {
+  border: 2px solid $createRigButtonBlue !important;
```

Comment:
this color name is super specific. try not to do this so that the colors remain reusable. I see the same setup in a few other places that we missed so we can either create a bug to fix them or just make sure we don't repeat in the future.

Reply:
sure thing